package.json

```diff
@@ -72,6 +72,7 @@
         "reselect": "^4.0.0",
         "rrweb": "^0.9.11",
         "sass": "^1.26.2",
+        "ts-node": "^9.1.1",
```

Comment:
My only hold-up is why this isn't kept as a dev dependency instead of a regular dependency? Tested locally and everything seems to work as expected too.

Reply:
Thanks for catching this! Should've been be a dev dependency